scripts/alternat.sh

```diff
@@ -190,10 +191,39 @@ install_cloudwatch_agent() {
    fi
 }
 
-# alterNAT config file containing inputs needed for initialization
-CONFIG_FILE="/etc/alternat.conf"
-
-load_config
+ASG_LIFECYCLE_HOOK_NAME="NATInstanceLaunchScript"
+complete_asg_lifecycle_action() {
+  if [[ -z "$1" ]]; then
+    echo "No lifecycle action result given"
+    return 2
```

Comment:
What is the reason for returning a value? If we don't use it, I think we can omit it and just `return`.